app/components/TuneCard/tests/index.test.js

```diff
@@ -0,0 +1,54 @@
+/**
+ *
+ * Tests for TuneCard
+ *
+ */
+
+import React from 'react';
+import { fireEvent } from '@testing-library/dom';
+import { renderWithIntl, timeout } from '@utils/testUtils';
+import TuneCard from '../index';
+
+describe('<TuneCard />', () => {
+  it('should render and match the snapshot', () => {
```

Comment:
add a test to make sure that everything is artistName, collectioName is getting rendered correctly

Reply:
Added in next commit